projects/Unstructured Supplemenrary  Service Data/ussdtim.py

```diff
@@ -0,0 +1,194 @@
+    print (gen_bvn)
+       				
+def open_acct( ):
+	global gen_bvn
+	print("Welcome to our online Account opening services.")
+	print("loading...")
+# creating an empty list to serve as a temporary place holder.
+	temp_storage= [ ]
+	f_name= input("Enter your first name:")
+	s_name= input ("Enter your second name:")
+	sex = input("Enter sex [M/F]:")
+	BVN_checker( )
+	temp_storage.append(f_name)
+	temp_storage.append(s_name)
+	temp_storage.append(sex)
+	temp_storage.append(gen_bvn)
+	details= " ".join(temp_storage)
+	print(details)
+	print("1. Press # to go back to options menu\n2. Press * to exit")
+	bck=input(":")
+	if bck=='#':
+	    options_menu( )
+	else:
+	    sys.exit( )
+	
+	exit( )
+	
+def upgrade_migrate( ):
+		print("Welcome to our online Upgrade/Migration services.\n 1. Ugrade\n 2. Migrate")
+		print("press # is go back to the Main Menu.")
+		prompt = int(input("Enter preferred Choice:"))
+		if prompt==1:
+			time.sleep(5)
+			print("Upgrading...")
+			exit( )
+		elif prompt == 2:
+			time.sleep(5)
+			print("Migrating...")
+			exit( )
+		elif prompt == "#":
+			options_menu( )
+		else:
+			sys.exit( )
+
+def balance ( ):
+	print("ACCOUNT\tBALANCE\n CHECKER")
+	print("press # is go back to the Main Menu.")
+	pin=input("Enter your 4 digit pin:")
+# isdigit( ) is used to  check for digits within a str while the nested if is used to make sure the user inputs 4 digits.
+
+###```i am to put the pin trial in a while loop```###REMINDER!!!
+
+	if len(pin)!=4:
+	   	print("Make sure its a 4digit pin.")
+	   	time.sleep(5)
+	   	balance( )
+	else:
+	   		    if pin.isdigit( )==True:
+	   		        time.sleep(5)
+	   		        print("Loading...")
+	   		        exit( )
+	   		    elif pin== "#":
+	   		        options_menu( )
+	   		    else:
+	   		        time.sleep(15)
+	   		        print("wrong pin")
+	   		        sys.exit( )
+		
+def transf( ):
+	print("1. Transfer self\n2. Transfer others")
+	print("press # is go back to the Main Menu.")
+	trnsf=input(":")
+	if trnsf == "#" :
+	    options_menu( )
+	elif trnsf == "1":
+	    time.sleep(5)
+	    print("Sending...")
+	    exit( )
+	elif trnsf=="2":
+	    time.sleep(5)
+	    num=int(input("Enter receivers mobile number:"))
+	    print("Transferring to",num)
+	    exit( )
+	else:
+	    if trnsf.isdigit( )!= True:
+	        time.sleep(5)
+	        print("Not an option")
+	        sys.exit( )
+	    elif trnsf.isdigit( )==True and len(trnsf)>2:
+	        time.sleep( 5)
+	        print("wrong password.")
+	        sys.exit( )
+	    else:
+	        time.sleep(10)
+	        print("An error has occurred")
+	        sys.exit( )
+	        	
+def funds( ):
+	time.sleep(3)
+	print(bank_list)
+	bnk = input("Select receipients Bank:")
+	acc_num= input("Entet account number:")
+	print("Sending to",acc_num)
+	hash= input("1.Press # to go back to options menu\n2. Press * to go exit.")
+	if hash == "#":
+	   options_menu( )
+	elif hash == "*":
+	    exit( )
+	else:
+														sys.exit( )											

```

Comment:
extra space ?
```suggestion
sys.exit( )											

```

Reply:
Makes the  code neater ,I don't want it all muddled up